src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIInteract.java

```diff
@@ -129,6 +132,36 @@ protected final boolean mineBlock(@NotNull final BlockPos blockToMine, @NotNull
         return true;
     }
 
+    private void triggerMinedBlock(@NotNull final BlockPos blockToMine)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "triggerMinedBlock" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)